### PR TITLE
Fix use of new constant from fastlane

### PR DESCRIPTION
**Pull request by jpeg-ms**

### CODE REVIEW

**Comment by jpeg-ms** [comment]
This fixes issues introduced by #169. My bad for not checking closely in which version of fastlane this new constant was introduced.

File: .rubocop.yml [hunk 1]
```diff
@@ -69,10 +69,10 @@ Lint/UselessAssignment:
     - '**/spec/**/*'
 
 # We could potentially enable the 2 below:
-Layout/IndentFirstHashElement:
+Layout/FirstHashElementIndentation:
```
Comment (phillipleblanc) [inline .rubocop.yml]: Looks like this PR includes changes from #171 ?

Comment (jpeg-ms) [reply .rubocop.yml]: Yeah it did. Not a problem anymore since #171 is merged :D

**Pull request merged**